src/components/GoalsList/GoalsList.js

```diff
@@ -78,7 +78,7 @@ const GoalsList = ({ setLoading, setAlert, topics, isTeamCalendar }) => {
                 {newGoalCard ? <NewGoalCard topics={topics} setNewGoalCard={setNewGoalCard} addGoal={addGoal} topic={topic} setTopic={setTopic} /> : ''}
                 {
                     goals.sort((goal1, goal2) => new Date(goal1.lastUpdated).getTime() > new Date(goal2.lastUpdated).getTime() ? -1 : 1).map(goal =>
-                        <GoalCard goal={goal} updateGoal={updateGoal} />
+                        <GoalCard goal={goal} updateGoal={updateGoal} isTeamCalendar={isTeamCalendar} />
```

Comment:
when you use map function you have to pass key prop to the element for example
GoalCard key={index} goal={goal} updateGoal={updateGoal} isTeamCalendar={isTeamCalendar} 
if you don't pass key you get warning, please check this you can see more warnings in browser console related to this issue :)